spec/dynamoid/transaction_write/update_spec.rb

```diff
@@ -0,0 +1,192 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require_relative 'context'
+
+# Dynamoid.config.logger.level = :debug
+
+describe Dynamoid::TransactionWrite do
+
+  describe '.execute' do
+    include_context 'transaction_write'
+
+    context 'updates' do
+      context 'simple primary key' do
+        before do
+          klass.create_table
+        end
+        it 'with attribute outside transaction' do
+          obj1 = klass.create!(name: "one")
+          obj1.name = "oneone"
+          Dynamoid::TransactionWrite.execute do |txn|
+            txn.update! obj1
```

Comment:
`update` with instance has different signature:

https://github.com/Dynamoid/dynamoid/blob/398f0657c68ef42aa41fa995f8a137ba6096d0ad/lib/dynamoid/persistence.rb#L748-L750

It's OK to skip this method for now.

Reply:
OK, skipping support for blocks in update() for now. I would like to support increment eventually.